internal/controllers/constants/constants.go

```diff
@@ -25,7 +25,7 @@ const (
 
 	// CSI Controller container and deployment names
 	TopolvmControllerDeploymentName = "topolvm-controller"
-	TopolvmControllerContainerName  = "topolvm-controller"
+	TopolvmControllerContainerName  = TopolvmControllerDeploymentName
```

Comment:
We can just delete `TopolvmControllerContainerName` and use `TopolvmControllerDeploymentName` instead.